### ISSUE TITLE
Network Switcher - Remaining Parts

### ISSUE THREAD

**Issue by michaeltomasik** [LiskHQ/lisk-hub#1982]
1. Used custom node address is saved only after login, should be saved on successful connect
1. Would be nice to have Enter working as Connect click in custom node address field
1. What should happen on app start if the last used network was custom node which is not available at the moment and connection failed?

**Comment by Efefefef** [comment]
@yasharAyari Please specify the expected behavior for 3.

**Comment by yasharAyari** [comment]
@Efefefef it needs to switch to `mainnet` when the custom node isn't available anymore.

**Comment by Efefefef** [comment]
@yasharAyari Isn't it risky?

**Comment by reyraa** [comment]
If the user tries to login, we should show an error indicating the node is not reachable. If the user tried to explore as guest, s/he should see the same error. navigation to dashboard must not be available when the last saved node is not reachable anymore.
